src/haproxy/server.go

```diff
@@ -0,0 +1,39 @@
+package haproxy
+
+import (
+	"net"
+)
+
+// Listener is a struct for providing a `net.Listener` compatible struct that checks for PROXY headers on new connections.
```

Comment:
Wrap long doc lines.  Not sure if you use VSCode, but if so, I use an extension called [Rewrap](https://github.com/stkb/Rewrap) which works pretty well.

Reply:
All wrapped up.